### DISCUSSION TITLE
Fade image transition v4

### DISCUSSION THREAD

**Discussion by Konstantintk** [fancyapps/ui#310] [Q&A]
Hello,

I attempted to look around for this feature in the documentation but wasn't able to find anything. I remember that earlier versions (v3 I think) had the option to change the slide transition from a "slide" to a "fade" when you pressed the next, previous, thumbnails, or arrow keys. However, it would still slide when you click and dragged the content; which was great for mobile devices. The closest thing I found was the "showClass: false" and "hideClass: false" options but that just abruptly changes one image to another as opposed to fading from one to another. Also this option only affects the thumbnails, and not any of the other keys. If you add the "friction: 0," on-top of that, the buttons will no longer cause the content to slide but it causes the images to pop or stick when you try to click and drag them. Is there a way to implement this effect in the new version? 

Thanks for the hard work and the amazing app.

**Comment by fancyapps** [comment]
Hi,

This feature has already been requested multiple times and will be available from v5.

**Reply by Konstantintk** [reply]
Great, thank you for the update
